release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* Use custom pipeline templates to create and deploy an application from a Git repository. These templates override the default pipeline templates provided by OpenShift Pipelines 1.5 and later.
+* Filter Helm charts based on their certification level and see all Helm charts in the *Developer Catalog*.
+* Use the options in the *Add* page to create applications and associated services and deploy these applications and services on {product-title}. The *Add* page options include: *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, and *Explore new developer features*.
+* Use workspaces in pipelines when creating applications in the pipeline builder. Add a trigger to a pipeline so you can use tasks to support workspaces in pipelines.
```

Comment:
- Use workspaces in pipelines when creating applications in the pipeline builder./Use workspaces when creating pipelines in the pipeline builder. - Pipelines builder helps you create/build pipelines not applications.
- You can also add a trigger...